src/nft/components/collection/Card.css.ts

```diff
@@ -22,14 +22,18 @@ export const loadingBackground = style({
 
 export const notSelectedCard = style([
   card,
-  sprinkles({
-    backgroundColor: 'backgroundSurface',
-    borderColor: 'backgroundOutline',
-    borderRadius: '14',
-  }),
   {
-    ':hover': {
-      backgroundColor: themeVars.colors.stateOverlayHover,
+    ':after': {
+      content: '',
+      position: 'absolute',
+      top: '0px',
+      right: ' 0px',
+      bottom: ' 0px',
+      left: '0px',
+      border: ' 1px solid',
+      borderRadius: '16px',
+      borderColor: '#5D678524',
+      pointerEvents: 'none',
```

Comment:
Clicking on `Details` now just selects the card instead of taking to the Details page and tooltips no longer show. I believe it may be caused by setting pointerEvents to none here and in `selectedCard`

Reply:
Was on me, didn't have latest when testing